src/main/java/mate/academy/spring/service/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+
+    public CinemaHallResponseDto parseToResp(CinemaHall cinemaHall) {
+        CinemaHallResponseDto dto = new CinemaHallResponseDto();
+        dto.setId(cinemaHall.getId());
+        dto.setCapacity(cinemaHall.getCapacity());
+        dto.setDescription(cinemaHall.getDescription());
+        return dto;
+    }
+
+    public CinemaHall parseToModel(CinemaHallRequestDto dto) {
```

Comment:
```suggestion
    public CinemaHall toModel(CinemaHallRequestDto dto) {
```
As described in common mistakes. Please fix in all places